content/docs/Configuration/_index.md

```diff
@@ -560,446 +560,18 @@ The information about releases is retrieved from Bodhi and because of the
 cache and required availability on Copr, it might take a while to get the
 newest state.
 
-#### Supported jobs
-
```

Comment:
Can you please give users a hint about where to find it?

Maybe a bullet list with a link and a short description

Reply:
the jobs in the list are clickable :thinking: 

https://me.mfocko.xyz/packit-config-reorg/docs/configuration/#supported-jobs

Reply:
Nice!